docs/machine-learning/tutorials/index.md

```diff
@@ -11,4 +11,7 @@ The following tutorials enable you to understand how to use [ML.NET](../index.md
 - [Taxi fare predictor](taxi-fare.md): demonstrates how to apply a **regression** task using ML.NET.
 - [Iris clustering](iris-clustering.md): demonstrates how to apply a **clustering** task using ML.NET.
 
+> [!NOTE]
+> The tutorials demonstrate how to use the ML.NET API introduced in ML.NET 0.1. For information about new API introduced in ML.NET 0.6, see [ML.NET high-level concepts](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetHighLevelConcepts.md) and [ML.NET Cookbook](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md).
```

Comment:
Please reference the current api as the LearningPipeline, and add "the" between about and new.
-> The tutorials demonstrate how to use the ML.NET `LearningPipeline` API introduced in ML.NET 0.1. For information about the new API introduced in ML.NET 0.6, see [ML.NET high-level concepts]